projects/step-core/src/lib/components/array-filter/array-filter.component.scss

```diff
@@ -0,0 +1,3 @@
+@use '../../../../styles/core-mixins' as core;
```

Comment:
I think that absolute import is better, than relative here, because if you will decide to move component somewhere else in future, you should keep in mind to change this relative import too.
In case of absolute import, while component in the same project the import path will always be the same
`@use 'projects/step-core/styles/core-mixins' as core;`